Pipfile

```diff
@@ -6,6 +6,7 @@ verify_ssl = true
 [dev-packages]
 pytest = "*"
 requests-mock = "*"
+coverage = "*"
```

Comment:
I think I had added this during my work to look at code coverage. Since it's not currently in use I would remove it and revert the Pipfile.lock changes too. If you want to start using it (not a bad idea for CI) I would do that in a future PR.